style.css

```diff
@@ -0,0 +1,269 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  background: #f7f7f9;
+}
+
+ul {
+  padding: 0;
+}
+
+h1 {
+  margin: 0;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  color: #6070ff;
+}
+
+img {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+button:enabled {
+  border: 1px solid #7f8cff;
+  background: #fff;
+  color: #6070ff;
+}
+
+button:hover {
+  background: #6070ff;
+  color: #fff;
+  box-shadow: 0.1px 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+button:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+  background-color: #fff;
+  color: #5e6c84;
+}
```

Comment:
please check the sequence of these events it is not correct.

Reply:
Please kindly state the reason(s) for the required change in a more friendly and polite manner. It might be obvious to you and not so for others. 👍